addons/web/static/src/css/base.sass

```diff
@@ -2124,9 +2124,10 @@ $sheet-padding: 16px
             padding: 0 !important
             cursor: ns-resize
             .oe_list_handle
-                font-size: 18px
+                font-size: 7px
                 color: #E0E0E0
-                margin-right: 7px
+                margin-right: 10px
+                margin-top: 8px
```

Comment:
Changing the vertical alignment will also change the vertical alignment in all the numerical columns, see the attached screenshot

![oe_number](https://cloud.githubusercontent.com/assets/8435879/5919384/9958a102-a633-11e4-8b99-fa47964a937c.png)